README.md

```diff
@@ -4,5 +4,5 @@ Commit 1<br>
 Commit 2<br>
 Commit 3
 
-
+I've been looking for pizza for several hours.
```

Comment:
You need to make a text bold on "pizza" word

Reply:
Hello, i'm just changing my code, you may check